CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ### New features
 
+#### We've adjusted our responsive type scale
+
+We've made the following adjustments to our responsive type scale:
+
+- point 16 is now 16px across all screen sizes
+- point 19 is now 19px across all screen sizes
+- point 24 on small screens is now font size 21px instead of 18px and line height 25px instead of 20px
```

Comment:
Just spotted this and my brain objected to 24 now being 21px 😮

But then it took me a few seconds to realise that 24 is still 24px, it's only on small screens that it's changing

Worth clarifying?

```suggestion
- point 24 stays at 24px for tablet users, but for small screens is now font size 21px instead of 18px and line height 25px instead of 20px
```

Reply:
I've made a minor adjustment to the suggested content but the sentiment to spell it out is good. Lemme know what you think. I'll bring in a tech writer to spruce it up once I've got code review as well.

Reply:
Sounds good, I'll have a look when it comes through

Reply:
I forgot to push it 😅 Done now.